lib/gallactickeys.js

```diff
@@ -96,6 +96,25 @@ gkeys.recover = (password, keyObject) => {
   return verifyAndDecrypt(derivedKey, iv, ciphertext, algo, ksCrypto.mac);
 }
 
+/**
+ * Get account information 
+ * @param {String} privateKey of the account
+ * @returns {Object} [contains privateKey, publicKey, accountAddress and validatorAddress]
+ */
+gkeys.inspectAccount = (privateKey) => {
+  let account = {};
+
+  const publicKeyHex = crypto.getTmPubKeyByPrivKey(privateKey);
+  const publicKey = crypto.bs58Encode(publicKeyHex, 4);
+  const address = crypto.getAcAddrByPrivKey(privateKey)
+
+  account.privateKey  = privateKey;
+  account.publicKey = publicKey;
+  account.address = address;
```

Comment:
we need to include Validator address here as well, like described in the `returns` inline comment. Also assign it under account field

Reply:
sure !